peering.tf

```diff
@@ -0,0 +1,28 @@
+module "peering" {
+  source = "github.com/ukhomeoffice/dq-tf-peering"
+
+  providers = {
+    aws = "aws.APPS"
+  }
+
+  cidr_block                     = "10.3.0.0/16"
+  haproxy_subnet_cidr_block      = "10.3.0.0/24"
+  connectivity_tester_subnet     = "10.3.2.0/24"
+  peering_connectivity_tester_ip = "10.3.2.11"
+  az                             = "eu-west-2a"
+  name_prefix                    = "dq-"
+
+  SGCIDRs = [
+    "10.1.0.0/16",
```

Comment:
don't these blocks come from elsewhere? can you not reference them by name?
i'd like to be able to read this and make sense of it without having to search for usages of the block.
also mean when it changes we only need to do it in one place